test/functionalTest/cases/0000_content_related_headers/missing_content_type_header.test

```diff
@@ -0,0 +1,111 @@
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Not really a new file, but a wrongly detected modification + mv.

NTC (just informative)